lib/ws-channel.js

```diff
@@ -0,0 +1,251 @@
+module.exports = WebSocketChannel;
+
+var assert = require('assert');
+var crypto = require('crypto');
+var debug = require('./debug')('ws-channel');
```

Comment:
I take back the `should be ('strong-control-channel:ws-channel')`, sorry, didn't see the `./debug`, which already does that.